tests/PySys/monitoring/monitoring_with_simulated_messages/run.py

```diff
@@ -102,7 +104,7 @@ def validate_json(self):
                     reason = "pressure stat validation failed in message: " + \
                         str(line)
                     self.abort(False, reason)
-        if self.time_cnt == 2 and self.temp_cnt == 1 and self.pres_cnt == 1:
+        if self.time_cnt == 1 and self.temp_cnt == 1 and self.pres_cnt == 1:
```

Comment:
I preferred to change the count of batches, rather than introducing a delay in the tests.

The behavior over a period of time is checked by `monitoring_smaller_interval`.